blog/2024-09-10-pants-2-22/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+  authors: [benjy]
+  tags: [announcement]
+  # The release happened on this date, despite what the file name implies (which is when this post was authored)
+  date: 2024-10-07
+---
+
+# Pants 2.22.0 is released!
+
+<CaptionedImg src={require("./splash.jpeg").default}>
+  Photo by{" "}
+  <a href="https://unsplash.com/@florianxrebmann?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">
+    Florian Rebmann
+  </a>{" "}
+  on{" "}
+  <a href="https://unsplash.com/photos/a-building-with-a-flag-on-top-cWAsvnD9A98?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">
+    Unsplash
+  </a>
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants [2.22.0](<(https://github.com/pantsbuild/pants/tree/2.22.x/src/python/pants/notes/2.22.x.md)>), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.22.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/docs/getting-started).
```

Comment:
I think this url should be https://www.pantsbuild.org/stable/docs/getting-started (or 2.22) to avoid hitting the old 2.18 redirect.

Reply:
Doen